src/resources.rs

```diff
@@ -58,6 +58,30 @@ enum Repr {
   TcpStream(tokio::net::TcpStream),
 }
 
+pub fn get_resource_table_entries() -> Vec<(i32, String)> {
+  let table = RESOURCE_TABLE.lock().unwrap();
+
+  let tuples = table
+    .iter()
+    .map(|(key, value)| (key.clone(), inspect_repr(&value)))
+    .collect();
+
+  tuples
+}
```

Comment:
Can you please add a rust test here. Here's a start:
```
#[test]
fn test_get_resource_table_entries() {
  assert_eq!(get_resource_table_entries().len(), 3);
  // Maybe add some other asserts?
}
```

Reply:
Right now I added TODO, I need some time to figure out how to run Tokio's functions in tests.

Reply:
I don't think the above test should work? I don't think you need to interact with tokio?